src/figuresClasses.ts

```diff
@@ -1,19 +1,73 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape:Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Input value should be more then 0');
+    }
+
+    if (!((a + b) > c && (a - b) < c)) {
+      throw new Error('Input value doesn\'t develop triangle');
+    }
+  }
 
-export class Circle {
+  getArea(): number {
+    const p: number = (this.a + this.b + this.c) / 2;
 
+    return Math.floor(Math.sqrt(p * (p - this.a) * (p - this.b) * (p - this.c))
+    * 100) / 100;
```

Comment:
you round down to hundredths on each getArea, maybe better create a separate method for it?